### PR TITLE
Fix bugs of project/add column listed by #215

### PR DESCRIPTION
**Pull request by siyuan0322**
Fix several bugs:
1. Dump schema when project.
2. Check existence before `get_edge_relationship`.
3. Load immediately after `add_column`.

related issue: #206 
related pr: #215

### CODE REVIEW

File: python/graphscope/framework/graph.py [hunk 1]
```diff
@@ -432,6 +432,7 @@ def add_column(self, results, selector):
         op = dag_utils.add_column(self, results, selector)
         graph = Graph(self._session, op)
         graph._base_graph = self
+        graph._ensure_loaded()
```
Comment (sighingnow) [inline python/graphscope/framework/graph.py:435]: Why `add_column` requires such force loading however `project` (as well as `project_to_simple`) doesn't need such hacking?

Comment (siyuan0322) [reply python/graphscope/framework/graph.py:435]: It's @acezen thinks that the error should be raised in the method, not later. and `_project_to_simple` does have some checking as it has more information in the python side.

Comment (sighingnow) [reply python/graphscope/framework/graph.py:435]: I see.

Leave a note/comment here would be better.

**Comment by acezen** [comment]
LGTM

**Review by acezen — APPROVED** [review]

**Pull request merged**